.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -0,0 +1,12 @@
+---
+name: Request a feature
+about: New ideas & improvements to Cesium are welcome.
+---
+
+<!-- 
+	Thanks for helping make Cesium better! 
+
+	When suggesting an idea, give examples of the intended use case.
+
+	If you're interested in seeing this feature in Cesium and want to work on it yourself,  we absolutely appreciate contributions! Check out the contributor's guide (https://github.com/AnalyticalGraphicsInc/cesium/blob/master/CONTRIBUTING.md). We're happy to provide advice and feedback. 
```

Comment:
For framing - why would someone want to contribute? They can get to it faster, fun, fame, etc.